From c2ef7cb36559cb2043d41ec732a1fc4757963dc0 Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Thu, 23 May 2019 15:18:02 -0700
Subject: [PATCH] TRITON-1689 nightly tests failing with "No compute resources"
 available Reviewed by: Josh Wilsdon <josh@wilsdon.ca> Approved by: Josh
 Wilsdon <josh@wilsdon.ca>

---
 src/fw/etc/examples/vmadm_vm1 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/fw/etc/examples/vmadm_vm1 b/src/fw/etc/examples/vmadm_vm1
index d3a9c4ff..75c6123a 100644
--- a/src/fw/etc/examples/vmadm_vm1
+++ b/src/fw/etc/examples/vmadm_vm1
@@ -13,6 +13,7 @@
     ],
     "owner_uuid": "e6c73bd2-fae4-4e0a-af76-2c05d088b066",
     "ram": 128,
+    "cpu_cap": 100,
     "tags": {
         "blocksmtp": true
     },
-- 
2.21.0

